include/meson.build

```diff
@@ -100,7 +100,7 @@ conf_data.set('NEED_DBUS', build_systemd_logind or build_hal ? '1' : false)
 conf_data.set('CONFIG_WSCONS', host_machine.system() in ['openbsd', 'netbsd'] ? '1' : false)
 
 conf_data.set('HAVE_XSHMFENCE', xshmfence_dep.found() ? '1' : false)
-conf_data.set('WITH_LIBDRM', libdrm_required ? '1' : false)
+conf_data.set('WITH_LIBDRM', libdrm_required and libdrm_dep.found() ? '1' : false)
```

Comment:
if libdrm_required is true, but libdrm not found / too old - why do we even proceed here ?

this now looks like some silent switch black magic.
shouldn't we rather make it really explicit ?

IMHO, if something needs (new enough) libdrm then the build should break with some helpful error message.